osbs/api.py

```diff
@@ -274,8 +274,15 @@ def _get_existing_build_config(self, build_config):
         - metadata.labels.git-repo-name AND metadata.labels.git-branch are equal
         """
 
-        git_labels = [(key, build_config['metadata']['labels'][key])
-                      for key in self._GIT_LABEL_KEYS]
+        git_labels = {
+            'repo_labels': [(key, build_config['metadata']['labels'][key])
+                            for key in self._GIT_LABEL_KEYS]
+        }
+        try:
+            git_labels['git-dir-name'] = build_config['metadata']['labels']['git-dir-name']
+        except KeyError:
+            git_labels['git-dir-name'] = None
```

Comment:
Can just use `git_labels[...] = build_config['metadata']['labels'].get(...)` in that case, no need for try:/except:.